### PR TITLE
Feature/remove product

### PR DESCRIPTION
**Pull request by dendoesit**
Added modal to remove products from cart.

This is blocked until the cart design updates pr is merged.

## Checklist before requesting a review

- [x] I have performed a self-review of my code
- [ ] Will this be part of a product update? If yes, please write one phrase about this update.

### CODE REVIEW

File: apps/store/package.json [hunk 1]
```diff
@@ -7,7 +7,7 @@
     "build": "next build",
     "start": "node --require ./server-preload.js ./node_modules/.bin/next start -p $PORT",
     "test": "jest --passWithNoTests --coverage",
-    "lint": "TIMING=1 next lint",
```
Comment (alebedev) [inline apps/store/package.json:10]: There's cross-env nom package for that - ```cross-env TIMING=1 next lint```

Do you want to try it?

File: apps/store/src/components/CartPage/CartPage.tsx [hunk 2]
```diff
@@ -12,16 +13,34 @@ export const CartPage = () => {
...
+        </Heading>
+      </PageHeader>
       <CartList products={products} />
-      <PriceBreakdown currency="SEK" products={products} cost={cost} />
-      <Button fullWidth={true}>
-        <Link href={PageLink.cartReview()}>Check Out</Link>
-      </Button>
-    </>
+      <Footer>
+        <PriceBreakdown currency="SEK" products={products} cost={cost} />
```
Comment (alebedev) [inline apps/store/src/components/CartPage/CartPage.tsx:25]: Can we move default currency to different level?  Probably another task, just caught my eye now.  I doubt we want to have currency default at presentation level, does not seem scaleable

File: apps/store/src/components/CartCard/CartCard.tsx [hunk 3]
```diff
@@ -39,26 +44,53 @@ const RemoveButton = styled.button({
...
         <HeaderElement>
-          {title && <TitleElement>{title}</TitleElement>}
-          <RemoveButton>Remove</RemoveButton>
+          <TitleElement>{title}</TitleElement>
+
+          <CartNotification.Root>
+            <CartNotification.Trigger asChild>
+              <RemoveButton>Remove</RemoveButton>
+            </CartNotification.Trigger>
+            <CartNotification.Content Title={<Header>Remove insurance ?</Header>}>
```
Comment (alebedev) [inline apps/store/src/components/CartCard/CartCard.tsx:74]: What's our policy on localisation here?  When it's OK to hardcode English text?

**Review by alebedev — COMMENTED** [review]
Left some questions, none of them critical for the review

**Review by guilhermespopolin — COMMENTED** [review]
I'd suggest display the buttons in the center of the modal dialog

image

File: apps/store/src/components/CartCard/CartCard.tsx (hunk 3 above)
Comment (dendoesit) [reply apps/store/src/components/CartCard/CartCard.tsx:74]: I believe we will get to the point where we will pass lokalize text keys but for now English text should be fine for presentation.

File: apps/store/package.json (hunk 1 above)
Comment (dendoesit) [reply apps/store/package.json:10]: I have managed to make it work using this:     "lint": "export TIMING=1 || set TIMING=1 &&  next lint",
but will give the cross-env a try.

File: apps/store/src/components/CartPage/CartPage.tsx (hunk 2 above)
Comment (dendoesit) [reply apps/store/src/components/CartPage/CartPage.tsx:25]: yeah, will definetely be part of a different task. should probably take currency out of url maybe.

**Comment by dendoesit** [comment]
great catch @guilhermespopolin . updated.

**Review by guilhermespopolin — APPROVED** [review]

**Review by alebedev — APPROVED** [review]

**Pull request merged**